test/e2e/specs/editor/various/fullscreen-mode.spec.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Full Screen', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test.afterEach( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllPosts();
+	} );
+
+	test( 'should open the fullscreen mode from the more menu', async ( {
+		page,
+	} ) => {
+		// Open Options
+		await page.locator( ' [aria-label="Options"]' ).click();
```

Comment:
```suggestion
		await page.click(
			'role=region[name="Editor top bar"i] >> role=button[name="Options"i]'
		);
```

Suggestion: The "Options" can be a common name for the button, so let's be more specific by using the region.